client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -45,23 +47,22 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 							{ translate( 'Everything you need to build and grow your site from any device.' ) }
 						</h3>
 					</div>
-					<div className="go-mobile__app-badges">
-						{ showIosBadge && (
-							<AppsBadge storeName={ 'ios' } utm_source={ 'calypso-customer-home' }></AppsBadge>
-						) }
-						{ showAndroidBadge && (
-							<AppsBadge storeName={ 'android' } utm_source={ 'calypso-customer-home' }></AppsBadge>
-						) }
-					</div>
+					{ showBadge ? (
+						<div className="go-mobile__app-badges">
+							<AppsBadge
+								storeName={ badgeStoreName }
+								utm_source={ 'calypso-customer-home' }
+							></AppsBadge>
+						</div>
+					) : (
+						<Button
+							className="go-mobile__email-link-button is-link"
+							href="https://apps.wordpress.com/get?campaign=calypso-customer-home"
+						>
+							{ translate( 'Get the Jetpack app' ) }
+						</Button>
+					) }
 				</div>
-				{ ! showIosBadge && ! showAndroidBadge && ! isDesktopApp && (
```

Comment:
I've removed the `! isDesktopApp` conditional for the button, but let me know if there's a reason we still need to hide this on the desktop app.

Reply:
I agree that removing `! isDesktopApp` makes sense. With the updated design and other conditionals, there could be a scenario where _no button/link_ is displayed, which would be an odd experience. This change means the desktop link will be displayed if the user is not on a mobile device.